### ISSUE TITLE
Add workflow for handling blocked issues

### ISSUE THREAD

**Issue by rarkins** [waffleio/waffle.io#595]
GitHub doesn't natively handle blocked and blocking issue dependencies, apart from allowing you to reference another issue and having it updated when it's closed. However from a project management perspective it is essential to be able to filter blocked issues in and out and alto to be immediately aware once an issue becomes unblocked. i.e. today an issue can stay as blocked for a long time after all blocking issues are closed, because there is no automatic way for it to be unblocked or for watchers to be notified.

Waffle.io could greatly improve this process by:
1. Adding native capability for filtering out blocked issue (could be solved if the filter field supported the minus filter, e.g. "badge -blocked" to list all issues containing the word badge but which don't contain blocked)
2. Auto removing the blocked label off issues once all issues it references have been closed.

**Comment by AntoineInsa** [comment]
+1

**Comment by zbeekman** [comment]
Some sort of issue dependency syntax would be greatly appreciated. You could use ‘blocked’ or ‘requires’ or ‘depends’ to denote the issue dependency. Tasks like moving these issues from ‘backlog’ or ‘blocked’ column to a column like ‘ready' when all the dependencies have been resolved would be great. See also #803 #1125

**Comment by msimerson** [comment]
:+1:

**Comment by rarkins** [comment]
Duplicate/part of #803
